spec/factories/veteran_onboardings.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
Is this being used?

Reply:
Yes, on line 9 of veteran_onboardings_controller_spec.rb